server/store/datastore/init.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !cgo
```

Comment:
INFO: this is used by newer go versions - we keep the old one too to be backwards compatible